### DISCUSSION TITLE
Carla Simulator exits automatically

### DISCUSSION THREAD

**Discussion by kksinghal** [carla-simulator/carla#4548] [General]
Using ubuntu docker image with vnc server, (https://hub.docker.com/r/dorowu/ubuntu-desktop-lxde-vnc/)
Using the latest Debian installation process.

<img width="894" alt="Screenshot 2021-08-17 at 2 02 09 PM" src="https://user-images.githubusercontent.com/26023139/129692247-fbe7d41e-3048-4357-a76a-71dd1c5978fe.png">

Running, ./CarlaUE4.sh, the window does not open and the command instantaneously exits automatically. How to fix this?

**Comment by ReCodeLife** [comment]
I came across the same question, is this problem solved now?

**Reply by kksinghal** [reply]
Not yet. I am working on installing Carla on cloud for development

**Reply by ReCodeLife** [reply]
> Not yet. I am working on installing Carla on cloud for development

I'm the same, maybe I should see if there's a carlaue4 source code, which has been bothering me for a few days and hopefully someone can help me.

**Reply by kksinghal** [reply]
I am on M1 mac, so I want to use cloud services as my virtual desktop. But, AWS workspace is very expensive, that's why I am configuring EC2 to allow remote display. Is there any other way you know I can use Carla with display? 
Edit: I am also trying to install on AWS EC2 Windows RDP

**Comment by kksinghal** [comment]
I got it working in AWS EC2. Use Windows Server 2016 AMI, and Microsoft remote desktop to connect to it. Use it just, like any other desktop
